src/web/app/pages-static/index-page/index-page.component.html

```diff
@@ -113,7 +113,7 @@ <h2>Not for Profit:</h2>
     <h2>Well Received by the Academic Community:</h2>
     <p>
       TEAMMATES has been presented, and well-received, in education-related conferences such as the
-      <a href="http://conferences.computer.org/cseet/2011/CSEET_2011/Index.html" target="_blank" rel="noopener noreferrer">International Conference on Software Engineering Education and Training (CSEET)</a>,
+      <a href="https://conferences.computer.org/cseet/2011/CSEET_2011/" target="_blank" rel="noopener noreferrer">International Conference on Software Engineering Education and Training (CSEET)</a>,
```

Comment:
Not part of what is described in the issue title, but I will close one eye as this is still part of UAT fix